fluent-package/apt/systemd-test/test.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+
+if [ -z $1 ]; then
+    echo "Error: Need to specify VM name in the Vagrantfile."
+    echo "Ex.) $ ./test.sh ubuntu-focal"
+    exit 1
+fi
+
+vm=$1
+dir="/vagrant/fluent-package/apt/systemd-test"
+
+vagrant status $vm | grep -E "^${vm}\s+not created (.*)$"
+if [ $? -ne 0 ]; then
+    echo "Error: The VM already exists. Need to destroy it in advance with the following command."
+    echo "$ vagrant destroy $vm"
+    exit 1
+fi
+
+set -eu
+
```

Comment:
In other test files, `set -exu` is used. is there any reason to dare to use `set -eu`?

Reply:
Yes.

I don't feel much need to display the commands in this file because it just runs other test files, and this file itself does not test anything.

In addition, it will bother `echo` behavior.

Reply:
I got it,  thanks.